packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-basic-sign-in/main/nodejs/integrationsteps.md

```diff
@@ -1,6 +1,6 @@
 ### Your app displays the sign-in page
 
-Build a sign-in page that captures both the user's name and their password.
+Build a sign-in page that captures both the `username` and `password`.
```

Comment:
This isn't talking about an actual field name. It's telling the dev to build a sign-in page that captures a user's name and a user's password.